### PR TITLE
[jsk_perception] Add robot mask nodelet

### PR DESCRIPTION
**Pull request by mmurooka**
robot_mask

### CODE REVIEW

**Comment by mmurooka** [comment]
https://github.com/blodow/realtime_urdf_filter provides same function.

File: jsk_perception/include/jsk_perception/robot_to_mask_image.h [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    virtual void onInit();
+    virtual void subscribe();
+    virtual void unsubscribe();
+    virtual void infoCallback(const sensor_msgs::CameraInfo::ConstPtr& info_msg);
+    virtual void initSelfMask(const ros::NodeHandle& pnh);
+
+    ros::Subscriber sub_;
+    ros::Publisher pub_;
+
+    robot_self_filter::SelfMask<pcl::PointXYZ>* self_mask_;
```
Comment (garaemon) [inline jsk_perception/include/jsk_perception/robot_to_mask_image.h]: shared_ptr

Comment (mmurooka) [inline jsk_perception/include/jsk_perception/robot_to_mask_image.h]: fixed

**Comment by garaemon** [comment]
merge origin/master.

This program may be important for some users.

**Comment by garaemon** [comment]
@otsubo @mizmizo Check out this program

**Pull request merged**